models/marts/core/fct_orders.sql

```diff
@@ -17,7 +17,7 @@ final as (
         orders.customer_id,
         orders.order_date,
         orders.status,
-        nvl(sum(case when payments.status = 'success' then payments.amount/100 else 0 end), 0) as amount
+        nvl(sum(case when payments.status = 'success' then payments.amount else 0 end), 0) as amount
     from orders
     left join payments on payments.order_id = orders.order_id
     group by 1, 2, 3, 4

```

Comment:
[For a cleaner reading group by, I recommend checking out the dbt_utils.group by macro.](https://github.com/fishtown-analytics/dbt-utils#group_by-source)